### PR TITLE
Jar-In-Jar EventBus

### PR DESCRIPTION
**Pull request by shedaniel**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -230,6 +230,7 @@ repositories {
 		dirs 'jars'
 	}
 	mavenCentral()
+	maven { url 'https://dl.bintray.com/patchworkmc/Patchwork-Maven/' }
```
Comment (natanfudge) [inline build.gradle]: Click the "upload to jcenter" button so you can just put `jcenter()` and that will include both patchwork and maven central, which is cleaner and improves build performance

Comment (shedaniel) [reply build.gradle]: I currently have no access to this bintray, we need to ask coderbot.

Comment (natanfudge) [reply build.gradle]: @coderbot16

Comment (TheGlitch76) [reply build.gradle]: getting approved takes a couple business days, i think this should be merged now and deal with that later

**Pull request merged**